### PR TITLE
added: get product by id

### PR DESCRIPTION
**Pull request by AshKet01**
added route in index.js (api). for get product by product id

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -32,6 +32,11 @@ app.get('api/user', (req, res) => {
     return res.json(user_id)
 })
 
+app.get('/api/product', (req, res) => {
+    const {prod_id} = req.query;
```
Comment (MayurBhowad) [inline src/index.js:36]: is this product id is unique?
and it only returning that product id.

**Review by MayurBhowad — COMMENTED** [review]
does not implemented any functionalty!

File: src/index.js (hunk 1 above)
Comment (AshKet01) [reply src/index.js:36]: yes. product id should be unique.
working on this route

File: src/index.js [hunk 2]
```diff
@@ -34,6 +34,7 @@ app.get('api/user', (req, res) => {
 
 app.get('/api/product', (req, res) => {
     const {prod_id} = req.query;
+    //do something
```
Comment (AshKet01) [inline src/index.js:37]: here is goes functionality.

**Review by AshKet01 — COMMENTED** [review]
do some

File: src/index.js (hunk 2 above)
Comment (MayurBhowad) [reply src/index.js:37]: okay! pulling

**Pull request merged**